helm/YataiService/values.yaml

```diff
@@ -0,0 +1,110 @@
+# Default values for YataiService.
+
+# Database URL following RFC-1738, and usually can
+# include username, password, hostname, database name as
+# well as optional keyword arguments for additional
+# configuration
+# (leave empty for SQLite)
+db_url: {}
+
+# Base URL for storing saved BentoService bundle files,
+# this can be a filesystem path(POSIX/Windows), or an S3
+# URL, usually starts with "s3://"
+repo_base_url: {}
+
+# Yatai Service
+yatai:
+  replicaCount: 1 # only supports 1 at the moment
+  image:
+    repo: bentoml/yatai-service
+    version: 0.8.5
```

Comment:
Let's move this to `values.yaml` and default to latest

Reply:
This is the `values.yaml` file isn't it?

Reply:
Oops. My bad. Shouldn't look at this without food